Circles.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+
+namespace Figures
+{
+    class Circles : Shape
```

Comment:
I suggest creating a constructor here so we could avoid using the `Radius` property.
You'll pass radius into the constructor and you'll call `CircleCoordinates()` method directly from the constructor.

Reply:
and lets add `double radius` as parameter for the `CircleCoordinates `

Reply:
done